.github/.OwlBot.yaml

```diff
@@ -267,445 +485,447 @@ deep-preserve-regex:
-  - source: /google/cloud/bigquery/datapolicies/v1/cloud.google.com/go/bigquery/datapolicies/apiv1
-    dest: /bigquery/datapolicies/apiv1
-  - source: /google/cloud/bigquery/datapolicies/v1beta1/cloud.google.com/go/bigquery/datapolicies/apiv1beta1
-    dest: /bigquery/datapolicies/apiv1beta1
-  - source: /google/cloud/bigquery/datatransfer/v1/cloud.google.com/go/bigquery/datatransfer/apiv1
-    dest: /bigquery/datatransfer/apiv1
-  - source: /google/cloud/bigquery/migration/v2/cloud.google.com/go/bigquery/migration/apiv2
-    dest: /bigquery/migration/apiv2
-  - source: /google/cloud/bigquery/migration/v2alpha/cloud.google.com/go/bigquery/migration/apiv2alpha
-    dest: /bigquery/migration/apiv2alpha
-  - source: /google/cloud/bigquery/reservation/v1/cloud.google.com/go/bigquery/reservation/apiv1
-    dest: /bigquery/reservation/apiv1
-  - source: /google/cloud/bigquery/storage/v1/cloud.google.com/go/bigquery/storage/apiv1
-    dest: /bigquery/storage/apiv1
-  - source: /google/cloud/bigquery/storage/v1beta1/cloud.google.com/go/bigquery/storage/apiv1beta1
-    dest: /bigquery/storage/apiv1beta1
-  - source: /google/cloud/bigquery/storage/v1beta2/cloud.google.com/go/bigquery/storage/apiv1beta2
-    dest: /bigquery/storage/apiv1beta2
-  - source: /google/cloud/billing/v1/cloud.google.com/go/billing/apiv1
-    dest: /billing/apiv1
-  - source: /google/cloud/billing/budgets/v1/cloud.google.com/go/billing/budgets/apiv1
-    dest: /billing/budgets/apiv1
-  - source: /google/cloud/billing/budgets/v1beta1/cloud.google.com/go/billing/budgets/apiv1beta1
-    dest: /billing/budgets/apiv1beta1
-  - source: /google/cloud/binaryauthorization/v1/cloud.google.com/go/binaryauthorization/apiv1
-    dest: /binaryauthorization/apiv1
-  - source: /google/cloud/binaryauthorization/v1beta1/cloud.google.com/go/binaryauthorization/apiv1beta1
-    dest: /binaryauthorization/apiv1beta1
-  - source: /google/cloud/certificatemanager/v1/cloud.google.com/go/certificatemanager/apiv1
-    dest: /certificatemanager/apiv1
-  - source: /google/cloud/channel/v1/cloud.google.com/go/channel/apiv1
-    dest: /channel/apiv1
-  # - source: /google/devtools/cloudbuild/v1/cloud.google.com/go/cloudbuild/apiv1/v2
-  #   dest: /cloudbuild/apiv1/v2
-  - source: /google/devtools/cloudbuild/v2/cloud.google.com/go/cloudbuild/apiv2
-    dest: /cloudbuild/apiv2
-  - source: /google/cloud/clouddms/v1/cloud.google.com/go/clouddms/apiv1
-    dest: /clouddms/apiv1
-  - source: /google/cloud/tasks/v2/cloud.google.com/go/cloudtasks/apiv2
-    dest: /cloudtasks/apiv2
-  - source: /google/cloud/tasks/v2beta2/cloud.google.com/go/cloudtasks/apiv2beta2
-    dest: /cloudtasks/apiv2beta2
-  - source: /google/cloud/tasks/v2beta3/cloud.google.com/go/cloudtasks/apiv2beta3
-    dest: /cloudtasks/apiv2beta3
-  - source: /google/cloud/compute/v1/cloud.google.com/go/compute/apiv1
-    dest: /compute/apiv1
+  - source: /google/cloud/accessapproval/v1/cloud.google.com/go
+    dest: /
```

Comment:
Would it be possible to make this the default value for `dest` for Go, perhaps? So we don't need this for every entry. If that is a bigger change elsewhere, we can come back to it

Reply:
I checked the [OwlBot README](https://github.com/googleapis/repo-automation-bots/blob/main/packages/owl-bot) as well as [config-files.ts](https://github.com/googleapis/repo-automation-bots/blob/main/packages/owl-bot/src/config-files.ts), but I didn't see any support for default values.

Reply:
Sounds like a feature request! Do you think it would be good to do? If so, can you file a feature request against owlbot?

Of course we shouldn't block this PR on completion of that.